app/actions/index.js

```diff
@@ -276,3 +280,46 @@ export const listRemoteSurveys = () => (dispatch, getState) => {
       );
   });
 };
+
+export const initializeObservation = tags => dispatch =>
+  dispatch({
+    type: types.INITIALIZE_OBSERVATION,
+    tags
+  });
+
+export const updateObservation = tags => dispatch =>
+  dispatch({
+    type: types.UPDATE_OBSERVATION,
+    tags
+  });
+
+export const saveObservation = () => (dispatch, getState) => {
+  const { observation } = selectActiveObservation(getState());
+
+  dispatch({
+    type: types.SAVING_OBSERVATION,
+    observation
+  });
+
+  // TODO this is wrong
+  // observation currently looks like (still needs a point / associated feature):
+  // {
+  //   tags: {
+  //     highway: "residential",
+  //     name: "My Street",
+  //     surface: "paved"
+  //   }
+  // }
+  return osm.createObservation(observation, error => {
```

Comment:
@noffle I could use some guidance here. My observation currently looks like a set of k/v pairs (though w/ @sethvincent's help it will acquire a point / associated feature). The API seems to expect GeoJSON (we could do that, moving tags into `properties`, but the actual geometry will always be a point) as well as an OSM node (which should be a feature, thus also needing a type attribute).

We'll probably want to need to add metadata (authorship, device id, etc) + private tags (fields marked private).